pkg/kubelet/cm/devicemanager/plugin/v1beta1/stub.go

```diff
@@ -144,13 +149,33 @@ func (m *Stub) Start() error {
 	return nil
 }
 
+func (m *Stub) Restart() error {
```

Comment:
The work you're doing here looks nice at glance, but I'm a bit worried that this very-test specific behavior (which has no other known usecase outside testing, has it?) requires changes to the common stub code. Is it possible to have the same functionality, or a good enough approximation, only in application code?

Reply:
Good point. What I'm concerned about is that I'm not aware of obvious use case for this feature outside the sample device plugin, but I concur this is generic enough to be a worthy addition, and emphasize the requirement of handling restarts. Let's go with it.